src/overview/filters/reducer.js

```diff
@@ -9,15 +10,33 @@ const defaultState = {
     tags: [],
     domainsInc: [],
     domainsExc: [],
+    // Will contain **ID** only one list for now
+    lists: '',
 }
 
-const addFilter = filterKey => (state, value) => ({
-    ...state,
-    [filterKey]: [...state[filterKey], value],
-    showFilters: true,
-})
+const addFilter = filterKey => (state, value) => {
+    if (filterKey === 'lists') {
+        return {
+            ...state,
+            [filterKey]: value,
+        }
+    }
+
+    return {
+        ...state,
+        [filterKey]: [...state[filterKey], value],
+        showFilters: true,
```

Comment:
I think this `showFilters` state is only used for showing the old filter view. Like if I type in "#test" in the search bar to add a tag called "test", this reducer will get triggered and the old filter view will still show up. Already talked about removing the component check, which should fix this, but we might as well remove this state (unless it's being reused for something else?)

Reply:
Sure, I was going to remove all the unused redux stuff after moving and testing things.